The device being used is: cuda

Changed the code!!!!!!!!!!
[34m[1mwandb[0m: [33mWARNING[0m Calling wandb.login() after wandb.init() has no effect.
[34m[1mwandb[0m:   74 of 74 files downloaded.  
/home/danfossautonomypc/segmentation/wandb_segment/artifacts/test-dataset:v2
/home/danfossautonomypc/segmentation/wandb_segment/artifacts/test-dataset:v2/test_data/masks/
